server/index.js

```diff
@@ -1,17 +1,16 @@
-// const express = require('express');
-// const db = require('./db');
-// const app = express();
-// const PORT = 8080;
+const express = require('express');
+const db = require('../database/insertDatabase');
+const app = express();
+const PORT = 8080;
```

Comment:
Here is where you define your port number, please change it to a number !8080 or !3000